.gitignore

```diff
@@ -22,6 +22,8 @@ user/*
 ###################
 *.pyc
 __pycache__
+/doc/reference/api/
```

Comment:
Why are we adding this? I think when we changed the vivarium-core docs to be more hand-crafted we stopped auto-generating the api stubs